default.env

```diff
@@ -2,26 +2,21 @@
 TZ=Europe/Berlin  
 ALWAYS_UPDATE_ON_START=true
 MULTITHREAD_ENABLED=true
-COMMUNITY_SERVER=true
+STEAMCMD_VALIDATE_FILES=true
+
+### Backups
 BACKUP_ENABLED=true
 BACKUP_CRON_EXPRESSION=0 * * * *
 BACKUP_RETENTION_POLICY=false
 BACKUP_RETENTION_AMOUNT_TO_KEEP=30
-STEAMCMD_VALIDATE_FILES=true
-# Change this to manual if you want to edit your configs yourself
-SERVER_SETTINGS_MODE=auto 
-WEBHOOK_ENABLED=false
-WEBHOOK_URL="YOUR-URL-IN-HERE"
-WEBHOOK_START_TITLE="Server is starting"
-WEBHOOK_START_DESCRIPTION="The gameserver is starting"
-WEBHOOK_START_COLOR="2328576"
-WEBHOOK_STOP_TITLE="Server has been stopped"
-WEBHOOK_STOP_DESCRIPTION="The gameserver has been stopped"
-WEBHOOK_STOP_COLOR="7413016"
-WEBHOOK_INFO_TITLE="Info"
-WEBHOOK_INFO_DESCRIPTION="This is an info from the server"
-WEBHOOK_INFO_COLOR="2849520"
+
+### Server Settings
+# Change this to manual if you want to edit your configs directly on file
+SERVER_SETTINGS_MODE=auto
+# Engine.ini
 NETSERVERMAXTICKRATE=120
+# PalWorldSettings.ini
```

Comment:
What does this here?

Reply:
Just a placeholder for people to know that changing the settings below will affect his file, doesn't change the functionality of .env neither makes it unreadable. I think this is a nice to have for people that want to change the .env file locally to know what settings are referred too what.